design/003-IoT-Adoption.md

```diff
@@ -0,0 +1,65 @@
+
+## Motivation
+
+The main advantage in the invention sketched in section 1 above, is that we can have restricted devices directly participate in peer-to-peer transactions while retaining the security / trust guarantees of a 2nd layer state channel protocol: to enable that we reduce the functionality required to be implemented in the device / agent by separating setting up and funding the state channels from the transactions via the channel (see Figure 2 and Figure 3).
+
+## Details
+
+Separating the functionality to reduce the needed available implementation and communication on a device. In general, the idea of a watchtower [3] is to react if the corresponding entity is offline or loses connectivity. Targeting IoT devices, the general assumption is that these devices have limited capabilities in regard to connectivity. Therefore, the watchtower or a watching function will run separately.
+
+## Rationale
+
+<!-- Provide a discussion of alternative approaches and trade offs; advantages
+and disadvantages of the specified approach.  -->
+
+As shown in Figure 2, the following main components should be available as independent components:
+
+* Funding Component: This component will act in the open, register and settle phase of a state channel. It funds the channel during open phase with the agreed initial balance. Additionally, it will close a channel with the given balance and is able to withdraw funds at the end.
+* Watching Component: This component receives   through an API states for already established channels and monitors the smart contracts, that other participating entity in the channel is not publishing an older state. Otherwise, the Watching Component will publish the latest known one. Newer retrieved valid states are overriding older ones. Its functionality is mapped to the transact phase of a state channel, where every party is able to publish a current state to be settled. In addition, it can be informed to stop watching a given channel.
+* Transaction component: This component is used by devices to perform off-chain transactions with other devices and to communicate with the Funding Component and the Watching Component. This will be the main component for channel participants, as they will use it to agree on the initial state creation and then inform the Funding Component to open the channel. Afterwards it will use it for interaction with the other participants. During these off-chain transactions, it will inform periodically the Watching Component with an actual valid state. At the end, it will send the final state to the Funding Component to close and settle the established state channel.
+
+## Figures
+
+![Current scenario](003/current-structure.svg)
+Figure 1: Current scenario: State channel nodes handle both on-chain  and off-chain transactions.
```

Comment:
You could insert a new line before the caption so that it formats more nicely. Same for Figure 2.

Reply:
will add it